source/features/clean-conversation-headers.css

```diff
@@ -40,11 +41,13 @@
 /* Removes on issues: [octocat] opened this issue on 1 Jan · 1 comments */
 /* Removes on PRs: [octocat] merged 1 commit into master from feature */
 .rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author + .Label, /* #5832 */
-.rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author {
+.rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author .author,
+.rgh-clean-conversation-headers.rgh-clean-conversation-headers-hide-author a[data-hovercard-url] {
 	display: none;
 }
 
-.rgh-clean-conversation-headers .commit-ref {
+.rgh-clean-conversation-headers .commit-ref,
+.rgh-clean-conversation-headers [class^='BranchName'] {
 	font-size: 12px;
 }
```

Comment:
Might be worth nesting selectors here.

```css
.rgh-clean-conversation-headers {
	.rgh-clean-conversation-headers-hide-author {
		.author+.Label,
		.author {
			
		}
	}
	.commit-ref,
	[class^='BranchName'] {
		
	}
}
```

Reply:
Not necessary though, it can be done separately since it affects the whole file.